metricbeat/docs/fields.asciidoc

```diff
@@ -59864,7 +59864,7 @@ type: keyword
 *`system.filesystem.files`*::
 +
 --
-The total number of file nodes in the file system.
+Total number of inodes on the system, which will be a combination of files, folders, symlinks, and devices.
```

Comment:
Is this obvious in the code doing this as well? Should we add a comment to the code itself calling this out?